src/ui/popup/manage-settings-page/export-theme.tsx

```diff
@@ -0,0 +1,35 @@
+import {m} from 'malevic';
+import {ViewProps} from '../types';
+import {Button} from '../../controls';
+import {saveFile} from '../../utils';
+import ControlGroup from '../control-group';
+
+export default function ExportTheme(props: ViewProps) {
+    chrome.runtime.onMessage.addListener(({type, data}) => {
```

Comment:
Add this listener inside `exportCSS` function. Inside a listener make it remove itself (`chrome.runtime.onMessage.removeListener`).